src/EventManager.cpp

```diff
@@ -29,6 +29,10 @@ void EventManager::updateMouse(sf::Vector2i mousePosition, std::vector<Platform>
 void EventManager::checkMouseOverPlatform(sf::Vector2i mousePosition, std::vector<Platform> &platforms){
 	int i = 0;
 	for (i; i < platforms.size(); ++i){
+		//If platform is not draggable, ignore
+		if (platforms[i].draggable == false){
```

Comment:
Put not draggable platforms in a different array

Reply:
Not happening on the fly in this pull request. Noted for later.